### PR TITLE
MB-1324 Add project overview to README

### PR DESCRIPTION
**Pull request by rswerve**
## Description

On Roci, we found that team members were answering the same questions every time a new engineer started on the project.  We decided to gather the highlights and write them up.  Please let me know if what we have here is accurate, sufficient, etc.

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-1324) for this change

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,6 +69,33 @@ Regenerate with "pre-commit run -a markdown-toc"
 
 <!-- tocstop -->
 
+## Overview
+
+MilMove is a system to help service members (and other authorized personnel) move their gear and possessions from one place to another.
+
+There are three React apps that compose the front end of the system:
```
Comment (chrisgilmerproj) [inline README.md]: You also want to talk about the mTLS apps: orders, contractors, gex, dps.

Comment (rswerve) [reply README.md]: Done.  Thanks to Erin for providing info, but please see if I got the translation right.

File: README.md [hunk 2]
```diff
@@ -73,29 +75,42 @@ Regenerate with "pre-commit run -a markdown-toc"
...
-Swagger has interactive documentation for users of the API, which allows us to test endpoints from within [the doc itself](#setup-milmovelocal-client)
+Swagger has interactive documentation for users of the API, which allows us to test endpoints from within [the doc itself](#setup-milmovelocal-client).
 
 Swagger also has a JS client that generates a set of API functions that are hooked into the React apps giving them easier access to the data.
 
+### The APIs
+
+In addition to the front end interfaces, MilMove comprises a set of APIs that work over Mutual TLS:
+
+`Contractors`: The API "the Prime" will use to communicate with the system. When the Prime needs information about a move (e.g., when is it scheduled?) or needs to update information about a move (e.g., what was the total weight of items moved?), it will use the `Contractors` API. As of this writing, this API is described in `prime.yaml`, but expect that name to change.
```
Comment (chrisgilmerproj) [inline README.md]: The contracts don't really use the word "Prime" and references to it are going to be removed from the codebase soon. So perhaps we just say "GHC"? Maybe @tinyels can weigh in there.

File: README.md [hunk 3]
```diff
@@ -73,29 +75,42 @@ Regenerate with "pre-commit run -a markdown-toc"
...
+
+`Orders`: People moving receive orders to move, which include information about the destination as well as authorizations for various kinds of storage, conveyance, per diem allowances, and many more details. Currently that's all transmitted via the Orders API in a PDF and has to be entered manually. Eventually the Orders API will allow all those details to be transmitted electronically.
```
Comment (chrisgilmerproj) [inline README.md]: This is slightly wrong.

```suggestion
`Orders`: People moving receive orders to move, which include information about the destination as well as authorizations for various kinds of storage, conveyance, per diem allowances, and many more details. Currently Orders information is entered into the system manually by office personnel from a PDF. However, the Orders API is receiving that data directly from the services and eventually that data will be used as the source of truth. That process is called "Orders Ingestion". The process of getting that data into the system for use with moves is called "Orders Integration".
```

File: README.md [hunk 4]
```diff
@@ -73,29 +75,42 @@ Regenerate with "pre-commit run -a markdown-toc"
...
 
-Go boilerplate code is generated from the Swagger files using a library called `go-swagger`. That code is automatically updated when a Swagger file is changed. These generated files are in `/pkg/gen`
+Go boilerplate code is generated from the Swagger files using a library called `go-swagger`. That code is automatically updated when a Swagger file is changed. These generated files are in `/pkg/gen`.
 
-Swagger has interactive documentation for users of the API, which allows us to test endpoints from within [the doc itself](#setup-milmovelocal-client)
+Swagger has interactive documentation for users of the API, which allows us to test endpoints from within [the doc itself](#setup-milmovelocal-client).
 
 Swagger also has a JS client that generates a set of API functions that are hooked into the React apps giving them easier access to the data.
 
+### The APIs
```
Comment (chrisgilmerproj) [inline README.md]: I wouldn't really say these are APIs. They are applications just like the above, only they don't have a UI. I think you add confusion if you say "The APIs" and fail to mention the internal API or the admin API. Perhaps I'd not distinguish between UIs and APIs this way. A better distinction is probably TLS vs mTLS applications.

Comment (rswerve) [reply README.md]: Thanks.  Made it TLS and Mutual TLS apps.

File: README.md (hunk 2 above)
Comment (rswerve) [reply README.md]: Removed mention of "The Prime"

**Pull request merged**